src/Compiler/Service/ServiceDeclarationLists.fs

```diff
@@ -875,17 +939,24 @@ module internal DescriptionListsImpl =
                 else FSharpGlyph.Class
             | Item.ModuleOrNamespaces(modref :: _) -> 
                   if modref.IsNamespace then FSharpGlyph.NameSpace else FSharpGlyph.Module
-            | Item.ArgName _ -> FSharpGlyph.Variable
+            | Item.NewDef _
+            | Item.ArgName _
             | Item.SetterArg _ -> FSharpGlyph.Variable
-            | _ -> FSharpGlyph.Error)
 
+            // These empty lists are not expected to occur
+            | Item.ModuleOrNamespaces []
+            | Item.UnqualifiedType [] ->
+                FSharpGlyph.Error
+            )
 
-    /// Get rid of groups of overloads an replace them with single items.
-    /// (This looks like it is doing the a similar thing as FlattenItems, this code 
-    /// duplication could potentially be removed)
-    let AnotherFlattenItems g m item =
+
+    /// Select the items that participate in a MethodGroup.  This is almost identical to SelectMethodGroupItems and
+    // should be merged, and indeed is only used on the 
+    let SelectMethodGroupItems g m item =
```

Comment:
Whole sentences are preferred, and it's either a comment // or a code comment ///

Reply:
Fixed, thanks